walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc

```diff
@@ -53,20 +55,20 @@ a *Realm*. The *Client* represents the application being secured and contains
 important details regarding the security applied to the application.
 
 . Navigate to the link:{sso-realm-url}[SSO Realm, window="_blank"].
-. Enter the username `{user-username}` and password `password` if prompted.
-. Details for the `{user-username}` realm are displayed upon login success.
+. Enter the username `walkthroughs` and password `password` if prompted.
+. Details for the `walkthroughs` realm are displayed upon login success.
 . Select *Clients* from the menu on the left.
 . Click the *Create* button at the top of the list of clients to display the *Add Client* screen:
-.. Enter `order-entry-system` in the *Client ID* field.
+.. Enter `{user-username}` in the *Client ID* field.
```

Comment:
I'd change this to `{user-username}-order-entry-system`. We might want them to add another. client in the future so having a better identifiers will be nice.

Reply:
created a new attribute 'client-name' and set it as you suggested, that avoids confusion, spotted a wrong user of {user-username} as a result ;)